### PR TITLE
:sparkles: Adding Top Pool and Coins tables 

### PR DESCRIPTION
**Pull request by KipandaJr**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206189181392452

### CODE REVIEW

File: views/metrics/top-tables/table.types.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export interface TableRowProps {
+  title?: string;
```
Comment (MarioBatalha) [inline views/metrics/top-tables/table.types.ts:2]: Must be required not optional

File: views/metrics/top-tables/top-coins-table/top-pools-table-head.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+import { TableHeadProps } from '../table.types';
+import TableRow from '../table-row';
+
+const TopPoolsTableHead: FC<TableHeadProps> = ({ title }) => {
+  return (
+    <>
+      <TableRow numCols={4} title={title} isTableHead>
+        <Typography variant="label" size="large" opacity={0.6}>
+          Coin
```
Comment (MarioBatalha) [inline views/metrics/top-tables/top-coins-table/top-pools-table-head.tsx:12]: This data could be dynamic

File: views/metrics/top-tables/top-pools-table/top-pools-table-head.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+import { TableHeadProps } from '../table.types';
+import TableRow from '../table-row';
+
+const TopPoolsTableHead: FC<TableHeadProps> = ({ title }) => {
+  return (
+    <>
+      <TableRow numCols={6} title={title} isTableHead>
+        <Typography variant="label" size="large" opacity={0.6}>
+          Token Pair
```
Comment (MarioBatalha) [inline views/metrics/top-tables/top-pools-table/top-pools-table-head.tsx:12]: This data also could be dynamic

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

**Comment by git-marcopitra** [comment]
@KipandaJr logic to fill the real values is on `sui-interface`, take a look

File: views/metrics/top-tables/table-head.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import { Typography } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+import { v4 } from 'uuid';
+
+import { TableHeadProps } from './table.types';
+import TableRow from './table-row';
+
+const TableHead: FC<TableHeadProps> = ({ title, columns }) => {
+  return (
```
Comment (git-marcopitra) [inline views/metrics/top-tables/table-head.tsx:9]: remove return

File: views/metrics/top-tables/table-row.tsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+}) => (
+  <>
+    {title && (
+      <Typography variant="title" size="large" fontSize="2rem" p="1.5rem">
+        {title}
+      </Typography>
+    )}
+    <Box
+      color="onSurfaceVariant"
+      px="2.25rem"
```
Comment (git-marcopitra) [inline views/metrics/top-tables/table-row.tsx:21]: If there is no this value on the theme use the near one

File: views/metrics/top-tables/top-coins-table/index.tsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import { Box } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import TableHead from '../table-head';
+import TopPoolsTableBody from './top-pools-table-body';
+
+const TopCoinsTable: FC = () => {
+  return (
```
Comment (git-marcopitra) [inline views/metrics/top-tables/top-coins-table/index.tsx:8]: remove the return

File: views/metrics/top-tables/top-coins-table/top-pools-table-body.tsx [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+const TopPoolsTableBody: FC = () => {
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setLoading(false);
+    }, 3000);
+  }, []);
+
+  return !loading ? (
```
Comment (git-marcopitra) [inline views/metrics/top-tables/top-coins-table/top-pools-table-body.tsx:20]: convert it to `if (loading) return`

File: views/metrics/top-tables/top-pools-table/index.tsx [hunk 8]
```diff
@@ -0,0 +1,38 @@
+import { Box } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import TableHead from '../table-head';
+import TopPoolsTableBody from './top-pools-table-body';
+
+const TopPoolsTable: FC = () => {
+  return (
```
Comment (git-marcopitra) [inline views/metrics/top-tables/top-pools-table/index.tsx:8]: Remove return

File: views/metrics/top-tables/top-pools-table/top-pools-table-body.tsx [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+const TopPoolsTableBody: FC = () => {
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setLoading(false);
+    }, 3000);
+  }, []);
+
+  return !loading ? (
```
Comment (git-marcopitra) [inline views/metrics/top-tables/top-pools-table/top-pools-table-body.tsx:20]: Convert it to `if (loading) return`

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

File: interface/sentio.ts [hunk 10]
```diff
@@ -0,0 +1,263 @@
...
+export interface CoinInfo {
+  symbol: string;
+  name: string;
+  decimals: number;
+  type: string;
+  bridge: null | Bridge;
+  sourceChain: Chain | null;
+}
+
+const COINS = {
```
Comment (git-marcopitra) [inline interface/sentio.ts:27]: move the `const`s to `constants` folder

File: package.json [hunk 11]
```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "@emotion/react": "^11.11.1",
     "@hookform/resolvers": "^3.3.2",
+    "@interest-protocol/sui-amm-sdk": "interest-protocol/sui-amm-sdk",
```
Comment (git-marcopitra) [inline package.json:18]: here is supposed to have the version

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

File: interface/sentio.ts [hunk 12]
```diff
@@ -0,0 +1,227 @@
...
+export interface CoinInfo {
+  symbol: string;
+  name: string;
+  decimals: number;
+  type: string;
+  bridge: null | Bridge;
+  sourceChain: Chain | null;
+}
+
+export const COINS_MAP = {
```
Comment (git-marcopitra) [inline interface/sentio.ts:27]: remove all the `const`

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Review by MarioBatalha — APPROVED** [review]

**Review by MarioBatalha — APPROVED** [review]

**Review by atilasousa — APPROVED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**